### DISCUSSION TITLE
`swr@0.5.5` was published on npm but can't be found on Github

### DISCUSSION THREAD

**Discussion by nickrttn** [vercel/swr#1087] [General]
`swr@0.5.5` was published 12 hours ago at the time of this post. However, the tag and its commits can't be found on the Github repo which makes me somewhat wary to install it. What's going on there?

**Comment by kylemh** [comment]
I think something exciting is in store: https://twitter.com/shuding_/status/1376281526900432896

**Reply by nickrttn** [reply]
Well, I would still like to know what is in `0.5.5` before upgrading to it :)

**Reply by shuding** [reply]
0.5.5 only adds #1085 as a hot-fix release. It was cherry-picked because on master we’re working on the next `beta` version.

**Reply by nickrttn** [reply]
Perfect, thanks :)
